.github/workflows/osv-scanner-scan.yml

```diff
@@ -0,0 +1,20 @@
+# This file is adapted from https://github.com/google/osv-scanner
+
+
+name: OSV-Scanner Scan
+
+on:
+  schedule:
+  - cron: "12 12 * * 1"
+
+# Restrict jobs in this workflow to have no permissions by default; permissions
+# should be granted per job as needed using a dedicated `permissions` block
+permissions: {}
+
+jobs:
+  scan-scheduled:
+    permissions:
+      contents: read # to fetch code (actions/checkout)
+      security-events: write # for uploading SARIF files
+    if: ${{ github.repository == 'metal3-io/ip-address-manager' && github.event_name == 'schedule' }}
```

Comment:
Based on this https://github.com/orgs/community/discussions/25641#discussioncomment-3248571, the `${{ .... }}` won't work when multiple conditions, it turns into ternary operator: https://docs.github.com/en/actions/learn-github-actions/expressions#example

Reply:
interesting it is working here https://github.com/google/osv-scanner/blob/main/.github/workflows/osv-scanner-unified-action.yml#L48, isn't the link you referred talking about multi line conditions in yaml or did I understand it differently ?

Reply:
lets keep it and see how it behaves

Reply:
Yeah documentation is not exactly clear, they only have simple examples so the if: can be intepreted in multiple ways. Let's test it out, then we'll know.